File: App-Preguntas-Respuestas/src/app/models/respuesta.ts
export class Respuesta {

    id?: number;
    descripcion: string;
    esCorrrecta: boolean;

    /**
     *
     */
    constructor(descripcion: string, esCorrecta: boolean, id?: number) {
        this.id = id;
        this.descripcion =descripcion;
        this.esCorrrecta = esCorrecta;

    }
}

File: BackEnd/BackEnd/Domain/Models/Respuesta.cs
using Microsoft.EntityFrameworkCore.Metadata.Internal;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace BackEnd.Domain.Models
{
    public class Respuesta
    {

        [Key]
        public int Id { get; set; }

        [Required]
        [Column(TypeName = "varchar(50)")]
        public string Descripcion { get; set; }
        [Required]
        public bool EsCorrecta { get; set; }
        
        public int PreguntaId { get; set; }
        public Pregunta? Pregunta { get; set; }


    }
}
